packages/atomic/src/components/product-listing/atomic-product-listing-interface/store.ts

```diff
@@ -22,66 +21,48 @@ export interface SortDropdownOption {
   label: string;
 }
 
-export interface AtomicStoreData extends AtomicCommonStoreData {
+export interface AtomicProductListingStoreData extends AtomicCommonStoreData {
   facets: FacetStore<FacetInfo>;
   numericFacets: FacetStore<FacetInfo & FacetValueFormat<NumericFacetValue>>;
   dateFacets: FacetStore<FacetInfo & FacetValueFormat<DateFacetValue>>;
   categoryFacets: FacetStore<FacetInfo>;
-  sortOptions: SortDropdownOption[];
-  mobileBreakpoint: string;
-  currentQuickviewPosition: number;
+  url: string;
 }
 
-export interface AtomicStore extends AtomicCommonStore<AtomicStoreData> {
-  getAllFacets(): {
-    [facetId: string]:
-      | FacetInfo
-      | (FacetInfo & FacetValueFormat<NumericFacetValue>)
-      | (FacetInfo & FacetValueFormat<DateFacetValue>);
-  };
-
-  isMobile(): boolean;
+export interface AtomicProductListingStore
+  extends AtomicCommonStore<AtomicProductListingStoreData> {
+  getFieldsToInclude(): string[];
+  setUrl(url: string): void;
+  getUrl(): string;
 }
 
-export function createAtomicStore(): AtomicStore {
-  const commonStore = createAtomicCommonStore<AtomicStoreData>({
+export function createAtomicProductListingStore(): AtomicProductListingStore {
+  const commonStore = createAtomicCommonStore<AtomicProductListingStoreData>({
     loadingFlags: [],
     facets: {},
     numericFacets: {},
     dateFacets: {},
     categoryFacets: {},
     facetElements: [],
-    sortOptions: [],
     iconAssetsPath: '',
-    mobileBreakpoint: DEFAULT_MOBILE_BREAKPOINT,
     fieldsToInclude: [],
-    currentQuickviewPosition: -1,
+    url: '',
   });
 
   return {
     ...commonStore,
-
-    getAllFacets() {
-      return {
-        ...commonStore.state.facets,
-        ...commonStore.state.dateFacets,
-        ...commonStore.state.categoryFacets,
-        ...commonStore.state.numericFacets,
-      };
-    },
-
-    isMobile() {
-      return !window.matchMedia(
-        makeDesktopQuery(commonStore.state.mobileBreakpoint)
-      ).matches;
-    },
-
     getUniqueIDFromEngine(engine: ProductListingEngine): string {
       return engine.state.productListing.responseId;
     },
+    setUrl(url: string) {
+      commonStore.set('url', url);
+    },
+    getUrl() {
+      return commonStore.get('url');
+    },
   };
 }
 
-export function makeDesktopQuery(mobileBreakpoint: string) {
-  return `only screen and (min-width: ${mobileBreakpoint})`;
-}
+// export function makeDesktopQuery(mobileBreakpoint: string) {
```

Comment:
Remove ?

Reply:
I kept it in case we need it in the future but might as well remove it for now.